internal/cmd/pipeline/command.go

```diff
@@ -80,27 +82,29 @@ func printTable(cmd *cobra.Command, pipeline streamdesignerv1.SdV1Pipeline) erro
 
 	if output.GetFormat(cmd) == output.Human {
 		table.Add(&humanOut{
-			Id:                  pipeline.GetId(),
-			Name:                pipeline.Spec.GetDisplayName(),
-			Description:         pipeline.Spec.GetDescription(),
-			KsqlCluster:         pipeline.Spec.KsqlCluster.GetId(),
-			SecretNames:         strings.Join(secrets, ", "),
-			ActivationPrivilege: pipeline.Spec.GetActivationPrivilege(),
-			State:               pipeline.Status.GetState(),
-			CreatedAt:           pipeline.Metadata.GetCreatedAt(),
-			UpdatedAt:           pipeline.Metadata.GetUpdatedAt(),
+			Id:                    pipeline.GetId(),
+			Name:                  pipeline.Spec.GetDisplayName(),
+			Description:           pipeline.Spec.GetDescription(),
+			KsqlCluster:           pipeline.Spec.KsqlCluster.GetId(),
+			SchemaRegistryCluster: pipeline.Spec.StreamGovernanceCluster.GetId(),
+			SecretNames:           strings.Join(secrets, ", "),
+			ActivationPrivilege:   pipeline.Spec.GetActivationPrivilege(),
+			State:                 pipeline.Status.GetState(),
+			CreatedAt:             pipeline.Metadata.GetCreatedAt(),
+			UpdatedAt:             pipeline.Metadata.GetUpdatedAt(),
 		})
 	} else {
 		table.Add(&serializedOut{
```

Comment:
are the contents of humanOut and SerializedOut different? if no, table.Print() can handle different formats of output as if you combine those 2 structs

Reply:
They're slightly different! I believe one formats timestamps differently than the other.